src/app/tabs/compileTab/compileTab.js

```diff
@@ -46,14 +46,16 @@ class CompileTab {
     this.compiler.setEvmVersion(this.evmVersion)
   }
 
+  setLanguage (lang) {
```

Comment:
What litteral can be `lang` ? "Solidity" or "solidity" or "sol" ? You may want to have a comment here : 
```
/**
 * ...
 * @params lang {'solidity' | 'yul' | 'vyper'} ...
 */
setLanguage (lang) 
```

Reply:
that can be `Solidity` or `Yul`.
@chriseth is there other valid input for the `language` property?

Reply:
Not for now.